pkg/manifests/yaml/rte/machineconfig.yaml

```diff
@@ -0,0 +1,33 @@
+# To update the SELinux policy you will need to decode the source content via base64, update it and encode back
```

Comment:
so it seems better to have it empty - or with a well-known and documented placeholder, and fill it programmatically.

Reply:
So we have two options:
1. save it under the const or variable under the code, IMHO it good enough 
2. save it under the file, we will need to copy it under the container image, and inject it dynamically during the start

Reply:
Let's go with the first option for now, seems good enough indeed